de.js

```diff
@@ -920,63 +920,63 @@ chall.name.dmg {0} Schaden zufügen {1}
 chall.name.bzk Erziele {0} Spielverderber-Abschüsse {1}
 chall.name.cc Erziele {0} hauchnahe Abschüsse {1}
 chall.name.rk Erziele {0} Vergeltungs-Abschüsse {1}
-event.doubleXP 2x Class XP enabled for this match
-ranked.join Joined Ranked Match. Abandoning a match will result in a temporary ban
-ranked.progress Game in Progress
-ranked.team Please join a Team
-ranked.switch Can't switch while in-game
-ranked.spectate Spectating Disabled in Ranked
-ranked.login Login to Spawn
+event.doubleXP 2x Klassen XP ein für dieses Match
+ranked.join Joined Ranked Spiel beigetreten. Ein ranked Match verlassen wird mit einem temporären Bann bestraft
+ranked.progress Spiel in Gange
+ranked.team Tritt einem Team bei
+ranked.switch Du kannst während der Runde deine Klasse nicht wechseln
+ranked.spectate Zuschauen ist im Ranked modus nicht gestattet
+ranked.login Logge dich ein um zu spawnen
 ranked.sync Missing Sync Data
-ranked.team.spawn Please join a Team to Spawn
-ranked.class.spawn Please select a Class to Spawn
-modes.round.win Your Team has secured this round
-modes.round.loss This round is no longer winnable
-modes.round.switch Switching Sides. Prepare for the next Round
-modes.trade.success Trade Successful
-modes.prophunt.limit [Prop Hunt] You are out of prop switches
-modes.prophunt.left [Prop Hunt] You have {0} switches remaining
-modes.prophunt.rotation.on [Prop Hunt] Rotation Enabled
-modes.prophunt.rotation.off [Prop Hunt] Rotation Disabled
-modes.prophunt.rotation.reset [Prop Hunt] Rotation reset.
-commands.login Login to use this command
-commands.connected {0} connected, {1} in menu
-commands.host {0} is the current Host.
-commands.noclip.on Noclip - Enabled
-commands.noclip.off Noclip - Disabled
-commands.god.on God mode - Enabled
-commands.god.off God mode - Disabled
-commands.god.on.other God mode - Enabled for {0}
-commands.god.off.other God mode - Disabled for {0}
-commands.smite.on Smite - Enabled
-commands.smite.off Smite - Disabled
-commands.smite.on.other Smite - Enabled for {0}
-commands.smite.off.other Smite - Disabled for {0}
-commands.perk Gave {0} {1}
+ranked.team.spawn Bitte tritt einem Team bei um zu spawnen
+ranked.class.spawn Bitte wähle eine Klasse aus um zu spawnen
+modes.round.win Dein Team hat diese Runde gesichert
+modes.round.loss Dein Team kann diese Runde nicht mehr gewinnen
+modes.round.switch Seitenwechsel Bereite dich auf die nächste Runde vor
+modes.trade.success Trade Erfolgreich
+modes.prophunt.limit [Prop Hunt] Du kannst dich nichtmehr verwandeln
+modes.prophunt.left [Prop Hunt] Du hast noch {0} Verwandlungen übrig
+modes.prophunt.rotation.on [Prop Hunt] Rotation ein
+modes.prophunt.rotation.off [Prop Hunt] Rotation aus
+modes.prophunt.rotation.reset [Prop Hunt] Rotation zurückgesetzt.
+commands.login Logge dich ein um diesen Befehl zu verwenden
+commands.connected {0} Verbunden, {1} im Menü
+commands.host {0} ist der momentanige Host.
```

Comment:
ist der derzeitige Host.

Reply:
"aktuelle" meets the meaning of the english word best imo